src/user/user.router.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-console */
+'use strict';
+
+const express = require('express');
+
+const userService = require('./user.service.js');
+
+const userRouter = express.Router();
+
+userRouter.get('/', (request, response) => {
+  const users = userService.getUsers();
+
+  response.send(users);
```

Comment:
move this logic to a controller. Same for all endpoints

Reply:
The router stands for the controller in this case. We do not need to add more files and split logic because it's already split enough.
We could also rename routes for controllers here but it will interfere with import naming.

What I was trying to say is that routes already handle all of the request/response operations and build as slim as it is possible.

Reply:
I agree with @DarkMistyRoom , because the point of this task is to understand how to build an MVC application, so please, create a controller for each entity and move this logic there as was mentioned above